packages/compile-vyper/index.js

```diff
@@ -216,7 +217,7 @@ const Compile = {
     // no vyper files found, no need to check vyper
     // (note that JSON-only will not activate vyper)
     if (vyperFiles.length === 0) {
-      return { compilations: [] };
+      return { compilations: [], contracts: [] };
```

Comment:
Not promoting this with the helper? 😁

(This may be worth a `Compilations.emptyWorkflowCompileResult()`, in seriousness)

Reply:
Oh, that's not a bad idea...

Reply:
OK, added this!